Signal/src/ViewControllers/ConversationView/Cells/OWSQuotedMessageView.m

```diff
@@ -315,6 +327,28 @@ - (nullable NSString *)fileTypeForSnippet
     return nil;
 }
 
+- (BOOL)isAudioAttachment
```

Comment:
So we use the mic for audio files *and* voice notes?

Reply:
Yes. We can't distinguish them here, right?